.github/workflows/julia-test-with-style.yml

```diff
@@ -33,9 +33,9 @@ jobs:
           make -j2
           LIBCEED_LIB=$(find $PWD/lib -name "libceed.*")
           pushd julia/LibCEED.jl
-          echo >> test/Project.toml
-          echo "[preferences.libCEED_jll]" >> test/Project.toml
-          echo "libceed_path = \"$LIBCEED_LIB\"" >> test/Project.toml
+          echo >> Project.toml
+          echo "[preferences.libCEED_jll]" >> Project.toml
+          echo "libceed_path = \"$LIBCEED_LIB\"" >> Project.toml
           [[ "$GITHUB_REF" =~ ^refs/(heads/release|tags/).* ]] || julia --project -e 'import Pkg; Pkg.test("LibCEED"; coverage=true, test_args=["--run-dev-tests"])'
-          git checkout test/Project.toml && julia --project -e 'import Pkg; Pkg.test("LibCEED")'
+          git checkout Project.toml && julia --project -e 'import Pkg; Pkg.test("LibCEED")'
```

Comment:
Are these changes still needed?

Reply:
Nope! Fixed in 833e5c2. Thanks @pazner  and @jeremylt!